R/carehomes.R

```diff
@@ -1508,6 +1516,57 @@ carehomes_parameters_observation <- function(exp_noise = 1e6) {
 }
 
 
+##' Carehomes observation parameters
+##'
+##' @title Carehomes sensitivity and specificity parameters
+##'
+##' @return A list of parameter values
+##'
+##' @param sero_specificity_1 Specificity of the first serology test assay
+##'
+##' @param sero_sensitivity_1 Sensitivity of the first serology test assay
+##'
+##' @param sero_specificity_2 Specificity of the second serology test assay
+##'
+##' @param sero_sensitivity_2 Sensitivity of the second serology test assay
+##'
+##' @param pillar2_specificity Specificity of the Pillar 2 test
+##'
+##' @param pillar2_sensitivity Sensitivity of the Pillar 2 test
+##'
+##' @param react_specificity Specificity of the REACT test
+##'
+##' @param react_sensitivity Sensitivity of the REACT test
+##'
+##' @export
+carehomes_parameters_sens_and_spec <- function(sero_specificity_1 = 0.9,
+                                               sero_sensitivity_1 = 0.99,
```

Comment:
I wonder if we should have called those high and low (for high and low performance) instead of 1 and 2 so that it's easier to know which one is assay 1 vs 2 but maybe too much of a pain

Reply:
I think I'd prefer to avoid hard-coding naming like that, in case e.g. we add a third assay. Learning from the experience of the likes of `beta_start` and `beta_end`!